### DISCUSSION TITLE
Save data from writer but not VTK

### DISCUSSION THREAD

**Discussion by Elphikaus** [Chaste/Chaste#177] [Q&A] [answered]
I am working with a cell-based off-lattice focusing on capsules project.
I want to perform some statistical modelling on the simulation data by running several thousand realisations.

Each time I run a simulation I get an associated output of several thousand files inside a `results_from_time_XX` folder.

Most of these files are related to VTK and corresponding visualisation in ParaView.

I have a couple of classes that will write specific data in a `.dat` file. These are all I need.

Broadly speaking is there any way to solve my simulation and not write anything to file other than my specific writer?

I know I have not provided anything specific, I can, but if there is like a general way to do this, that would be great.

**Comment by jmosborne** [comment]
If you turn VTK of it will not output the VTK files, for the other files all are output with a CellWriter CellPopulationWriter or  CellPopulationCountWriter so check you havent added any more of these. Some of these are included as default but you can stop these by calling SetOutputResultsForChasteVisualizeron your population.  This should remove a lot of the other files.

From: Mr ***@***.***>
Date: Wednesday, 10 May 2023 at 2:26 am
To: Chaste/Chaste ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Chaste/Chaste] Save data from writer but not VTK (Issue #97)

I am working with a cell-based off-lattice focusing on capsules project.
I want to perform some statistical modelling on the simulation data by running several thousand realisations.

Each time I run a simulation I get an associated output of several thousand files inside a results_from_time_XX folder.

Most of these files are related to VTK and corresponding visualisation in ParaView.

I have a couple of classes that will write specific data in a .dat file. These are all I need.

Broadly speaking is there any way to solve my simulation and not write anything to file other than my specific writer?

I know I have not provided anything specific, I can, but if there is like a general way to do this, that would be great.

**Comment by Elphikaus** [comment]
How do you turn Vtk off?

**Comment by kwabenantim** [comment, marked as answer]
Hi @fieldofnodes. You can turn VTK off by configuring Chaste with:

```
cmake -DChaste_USE_VTK=OFF /path/to/Chaste/source
```
